app/controllers/comments_controller.rb

```diff
@@ -47,4 +47,15 @@ def create
   def set_subjects
     @subjects = Comment::SUBJECTS
   end
+
+  def recaptcha_valid?
+    return true unless Settings.recaptcha.site_key.present?
+    options = case Settings.recaptcha.type
+              when "v2_checkbox"
+                { model: @comment }
+              when "v3"
+                { action: Settings.recaptcha.v3.action, minimum_score: Settings.recaptcha.v3.minimum_score }
```

Comment:
Do we need to pass in the model here? Not sure how v3 differs from v2.

Reply:
I think v3 only needs action and minimum_score. https://github.com/ambethia/recaptcha#verify_recaptcha-use-with-v3